tests/test_core.py

```diff
@@ -668,6 +668,22 @@ def test_interpolate(test_df_year):
     assert not test_df_year.filter().data.duplicated().any()
 
 
+def test_interpolate_datetimes(test_df):
+    # test that interpolation also works with date-times.
+    some_date = datetime.datetime(2007, 7, 1)
+    if test_df.time_col == "year":
+        with pytest.raises(ValueError):
```

Comment:
I think this can be `pytest.raises(ValueError, test_df, interpolate, time=some_date)` for saving a line...